packages/react-composites/src/composites/CallComposite/components/buttons/Custom.tsx

```diff
@@ -9,6 +9,17 @@ import {
   CustomCallControlButtonPlacement
 } from '../../types/CallControlOptions';
 
+/**
+ * Max number of Custom Buttons in primary and secondary ControlBar
+ *
+ * @private
+ */
+export const CUSTOM_BUTTON_OPTIONS = {
+  MAX_PRIMARY_DESKTOP_CUSTOM_BUTTONS: 3,
```

Comment:
These values exclude the existing buttons in the control bar?

Reply:
Yes, they exclude the existing buttons in the control bar. 
I will make that explicit in the description.